style.css

```diff
@@ -329,6 +329,75 @@ hr {
   margin-bottom: 76px;
 }
 
+.mobile-menu {
+  display: none;
+  width: 100%;
+  height: 100%;
+  background-color: rgb(25 23 101 / 100%);
+  position: fixed;
+  top: 0;
+  margin-top: 32px;
+  background-image: url(images/shapes\ mobile\ menu.png);
+  background-repeat: no-repeat;
+  background-position: 100%;
+}
+
+.mobile-menu .imag {
+  display: flex;
+  justify-content: flex-end;
+  margin-right: 43px;
+}
+
+.menu-list {
+  left: 100%;
+  min-height: 100vh;
+  transition: 0.4s;
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+}
+
+.menu-list a {
+  color: #edecff;
+  font-size: 24px;
+  font-family: "Roboto", sans-serif;
+  line-height: 32px;
+  text-decoration: none;
+}
+
+.menu-list li {
+  display: flex;
+  padding: 8px;
+  justify-content: space-around;
+}
+
+.hamburger div {
+  width: 30px;
+  height: 3px;
+  background-color: #edecff;
+  margin: 6px 0;
+  cursor: pointer;
+  transition: 0.4s;
+  margin-left: 24px;
+}
+
+.active .bar-1 {
+  transform: rotate(-45deg) translate(-7px, 5px);
+}
+
+.active .bar-2 {
+  opacity: 0;
+}
+
```

Comment:
- [ ] Kindly make sure the background image for the mobile menu is placed properly as per the Figma design.
- [ ] Kindly make sure that the menu options are correctly placed with correct spacing as per the Figma design.
##  Make sure to follow the video attached to see where you have to make changes and make sure your design resembles Figma's design.

https://user-images.githubusercontent.com/70269507/169270596-07c7ab8a-4db1-47d9-bb18-f41b296867a4.mov